include/hawkbit-client.h

```diff
@@ -90,6 +90,7 @@ typedef struct Artifact_ {
         gchar *download_url;          /**< download URL of software bundle file */
         gchar *feedback_url;          /**< URL status feedback should be sent to */
         gchar *sha1;                  /**< sha1 checksum of software bundle file */
+        gchar *do_maintenance;        /**< maintenance flag, possible values: available, unavailable, null */
```

Comment:
`do_maintenance` sounds like a boolean. Maybe `maintenance_window` is better and closer to the DDI API?

Reply:
It's actually 3 state like mentioned in the comments, `undefined` does not mean `false`. I'm OK to rename, but wouldn't it be confusing with Target API, which actually defines the window (not just availability).

Reply:
Hm, the Target API is part of the Management API which we don't implement anyway (apart from `script/hawkbit_mgmt.py`). So I'd opt for renaming it. The DDI API attribute calls it `deployment.maintenanceWindow`, so `maintenance_window` should be okay.

Reply:
Done